src/components/FilterTags/FilterTags.js

```diff
@@ -87,20 +88,95 @@ const FilterTags = () => {
     };
   }, []);
 
-  const handleFilterClick = () => {
-    console.log("Filter clicked");
+  /**
+  Apply a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const addActiveFilter = (e, filter, filterBy) => {
+    setActiveFilters(filters => {
+      const updatedFilters = { ...filters };
+      updatedFilters[filterBy] = updatedFilters[filterBy] || [];
+      if (!updatedFilters[filterBy].includes(filter)) {
+        updatedFilters[filterBy].push(filter);
+      }
+      return updatedFilters;
+    });
+    e.currentTarget.classList.add("is-selected");
   };
 
+  /**
+  Remove a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const removeActiveFilter = (e, filter, filterBy) => {
+    e.stopPropagation();
+    setActiveFilters(filters => {
+      const updatedFilters = {
+        ...filters
+      };
+      if (updatedFilters[filterBy].includes(filter)) {
+        const index = updatedFilters[filterBy].indexOf(filter);
+        updatedFilters[filterBy].splice(index, 1);
+      }
+      return updatedFilters;
+    });
+
+    // Remove is-selected state from relevant filter from filter panel
+    const panelFilterItems = document.querySelectorAll(
+      ".p-filter-panel__button"
+    );
+    panelFilterItems.forEach(panelFilterItem => {
+      if (panelFilterItem.innerHTML === filter) {
+        panelFilterItem.classList.remove("is-selected");
+      }
+    });
+  };
+
+  let activeFiltersCount = 0;
+  Object.values(activeFilters).forEach(filterGroup => {
+    activeFiltersCount = activeFiltersCount + filterGroup.length;
+  });
+
   return (
     <div className="p-filter-tags" ref={node}>
-      <form>
-        <input
-          type="text"
-          placeholder="Filter models"
-          className="p-filter-tags__input"
-          onFocus={() => setfilterPanelVisibility(true)}
-        />
-      </form>
+      <div
+        type="text"
+        className="p-filter-tags__input"
+        onClick={() => setFilterPanelVisibility(!filterPanelVisibility)}
+      >
+        {Object.entries(activeFilters).length > 0 &&
+          Object.entries(activeFilters).map(activeFilterObj => {
+            return Object.values(activeFilterObj[1]).map(activeFilter => {
+              return (
+                <span
+                  className="p-filter-tags__active-filter"
+                  key={activeFilterObj[0] + activeFilter}
+                >
+                  {activeFilterObj[0]}: {activeFilter}{" "}
+                  <i
+                    className="p-icon--close"
```

Comment:
QA Issue:
<img width="268" alt="Screen Shot 2020-01-27 at 11 22 34 PM" src="https://user-images.githubusercontent.com/532033/73237664-eea76a80-415b-11ea-828f-4cd70f72c04d.png">

The close button can end up under the overflow count. The 'input' should probably end before the count element to avoid this.